windows/RNSVG/SvgView.cpp

```diff
@@ -0,0 +1,194 @@
+#include "pch.h"
+
+#include "SvgView.h"
+#if __has_include("SvgView.g.cpp")
+#include "SvgView.g.cpp"
+#endif
+
+#include <winrt/Windows.Graphics.Display.h>
+
+#include "GroupView.h"
+#include "Utils.h"
+
+using namespace winrt;
+using namespace Microsoft::Graphics::Canvas;
+using namespace Microsoft::ReactNative;
+using namespace Windows::Graphics::Display;
+
+namespace winrt::RNSVG::implementation {
+SvgView::SvgView(IReactContext const &context) : m_reactContext(context) {
+  m_scale = static_cast<float>(DisplayInformation::GetForCurrentView().ResolutionScale()) / 100;
+
+  m_canvasDrawRevoker = m_canvas.Draw(winrt::auto_revoke, {get_weak(), &SvgView::Canvas_Draw});
```

Comment:
exceptions that come from CreateGeometry (e.g. a parsing error) would end up bubbling to this lambda, and probably killing the app. Can you add some error handling here to at least redbox/yellowbox?